docs/userGuide/contentAuthoring.md

```diff
@@ -113,6 +113,24 @@ In addition, it supports:
 
 * More media blocks, embedding services and additional options can be found in [Markdown-it-block-embed docs](https://github.com/rotorz/markdown-it-block-embed)
 
+### Add your own variables
+
+In `_markbind/variables.md`, you can define your own variables which can be used anywhere in your site. The format is as follows.
+
+```
+<span id="year">2018</span>
+
+<span id="options">
+* yes
+* no
+* maybe
+</span>
+```
+
+Each variable must have a id and the value can be any markbind-compliant code fragment.
+
+In any file in your site, you can include the variable by surrounding the variable's id with double curly braces e.g. <code>{&#8203;{options}&#8203;}</code>

```

Comment:
Mention what is acceptable as a variable name? I noticed that dashes are not allowed in variable names.

Reply:
nunjucks does a lot more than we currently need (i.e. substituting variables). For example, it supports math operations. Hence gly-search is not treated as a string/key. Instead, it is treated as the math minus symbol. (so i think there is some not a number error)

Mustache.js / Handlebars.js might work

Reply:
We can address that as a separate PR. For now, let's mention the allowed format so that users aren't surprised when they try to use `abc-def` type variables (it's not uncommon to use that kind of identifiers to be used in web programming).